.github/workflows/tfplugindocs-check.yml

```diff
@@ -0,0 +1,49 @@
+name: Check if tfplugindocs result matches /docs
+
+on:
+  push:
+    branches:
```

Comment:
we dont have branches named release/**.

I think having this workflow check if a PR has generated docs correctly is sufficient